packages/frontend/src/pages/Playground.svelte

```diff
@@ -124,13 +146,29 @@ function getSendPromptTitle(sendEnabled: boolean, status?: string, health?: stri
     title="{conversation?.name}"
     searchEnabled="{false}"
     contentBackground="bg-charcoal-500">
-    <svelte:fragment slot="subtitle">{model?.name}</svelte:fragment>
+    <svelte:fragment slot="icon">
+      <div class="mr-3">
+        <StatusIcon
+          icon="{ContainerIcon}"
+          size="{24}"
+          status="{getStatusForIcon(server?.status, server?.health?.Status)}" />
+      </div>
+    </svelte:fragment>
+    <svelte:fragment slot="subtitle">
+      <div class="flex gap-x-2 items-center">
+        {#if model}
+          <div class="text-xs" aria-label="Model name">Model: {model.name}</div>
+          <Chip
+            icon="{faMemory}"
+            content="RAM usage: {model.memory ? filesize(model.memory, { base: 2 }) : 'N/A'}"
```

Comment:
I am against displaying the ram memory **estimate** on the playground page. As a user I would think this would be the real ram usage of the model, when it is not.

We either need to remove this estimate of the header, or use the real value.

Reply:
removed